course-materials/discussion/week1-discussion.Rmd

```diff
@@ -63,6 +63,7 @@ suppressMessages({
 library(tidyverse)
 library(gt)
 library(openintro)
+library(here)
```

Comment:
Heads up for the future, if we use `here::here()` then `library(here)` isn't necessary. The double colon tells R to look for the `here()` function in the `here` package.